app/src/controllers/instance/organizations/constants.js

```diff
@@ -19,6 +19,7 @@ import { PAGE_KEY, SIZE_KEY } from 'controllers/pagination';
 export const NAMESPACE = 'organizations';
 
 export const FETCH_ORGANIZATIONS = 'fetchOrganizations';
+export const FILTERED_ORGANIZATIONS = 'filteredOrganizations';
```

Comment:
It is an action.
It should be named as `FETCH_FILTERED_ORGANIZATIONS.`
Let's leave the TODO comment to leave a single fetch action here and for org projects.

Reply:
Done.

Reply:
@BlazarQSO what about TODO comment?

Reply:
I don't understand what kind of comment is needed